dtale/app.py

```diff
@@ -755,15 +751,13 @@ def instances():
     """
     Prints all urls to the current pieces of data being viewed
     """
-    curr_data = global_state.get_data()
-
-    if len(curr_data):
+    if global_state.size() > 0:
```

Comment:
```suggestion
    if global_state.size():
```

Reply:
You can also fo `len(global_state)` since I see you implemented the `__len__` function

Reply:
I don't think it works because functions starts with _ are not exported. I could change it though but I was too afraid to accidentally overwrite module global variables.

Reply:
Ahh, very good point. Leave it